rpi3-convert-stage-5.sh

```diff
@@ -80,7 +118,6 @@ install_files() {
 
   sudo cp ${bin_dir_pi}/u-boot.bin ${boot_dir}/kernel7.img
   sudo cp ${bin_dir_pi}/boot.scr ${boot_dir}
-  sudo cp ${bin_dir_pi}/init_resize.sh  ${rootfs_dir}/usr/lib/raspi-config/init-resize.sh
```

Comment:
This change seems out of place, but otherwise it looks good to me.

Reply:
init_resize.sh was just installed/copied in two places - I removed surplus line.

Reply:
init_resize.sh was just installed/copied in two places - I removed surplus line. So if the rest is ok, then please resolve conversation.